src/components/Calculator.js

```diff
@@ -78,24 +79,42 @@ const keys = [
     class: 'btn orange',
   },
 ];
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
+const Calculator = () => {
+  const [prev, setState] = useState({ total: null, next: null, operation: null });
```

Comment:
- [ ] Great job so far your app works as expected 👍 but in this project, you are to use `state` and `props`, pass props to child components, use lifecycle methods, and lift state up. Instead of `props` you've used `hooks`. Kindly convert your `Calculator` component from a function back to a class and use `props` and `state` which should be initialized in the constructor since you are going to be using `hooks` in the upcoming project 👍